apps/staking/locales/en.json

```diff
@@ -1032,7 +1032,8 @@
       "confirmHeading": "Are you sure?",
       "confirmTextS1": "This will remove the current owner {ethAddress}",
       "confirmTextS2": "and transfer ownership and control of this vesting contract to the new owner {ethAddress}",
-      "confirmTextS3": "<br></br>All funds will belong to the new address. This action is irreversible and cannot be undone.",
+      "confirmTextS3": "<br></br>By proceeding, you confirm that:<br></br>\nYou own the new wallet address<br></br>You are NOT selling, transferring, granting any participation in, or distributing tokens",
```

Comment:
do we want
`<br></br>` instead of `</br>`
and also, do we want 
`<br>` and `\n` ?

Reply:
unfortunately self closing tags are not supported, and the /n doesnt do anything but ill remove the /n in crowdin